Assignment3/main.js

```diff
@@ -0,0 +1,31 @@
+var tools = require("./problem1.js");
+var array = [1, 7, 10, 3, 8];
+var value = tools.secondLargest(array);
+console.log("Program 1 --------");
+console.log("Array = ["+array+"]");
+console.log("Second Largest Value: "+value);
+console.log('\n');
+
+var tools = require("./problem2.js");
```

Comment:
you can't use require anywhere in the middle of the code. `require` are much like import statements in Java. Make sure to require/import everything you need at the beginning of the file